src/pages/programDisaggregations/form/CategoryMappingSection.tsx

```diff
@@ -152,11 +156,23 @@ const useCategories = ({
 
         // A suggested category is a category that is part of a selected categoryCombo
         // in a program indicator mapping, but not already mapped in the form.
+
+        // keep track of categories included to avoid duplicates
+        const alreadyIncludedInSuggestedCategories: { [key: string]: boolean } =
```

Comment:
Nice catch, and thanks for fixing this!

Minor, but I think I would've preferred a Set here though. 
```suggestion
     const alreadyIncludedInSuggestedCategories = new Set<string>()
```